released/packages/coq-unicoq/coq-unicoq.1.6+8.17/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "matthieu.sozeau@inria.fr"
+authors: [ "Matthieu Sozeau <matthieu.sozeau@inria.fr>" "Beta Ziliani <beta@mpi-sws.org>" ]
+dev-repo: "git+https://github.com/unicoq/unicoq.git"
+homepage: "https://github.com/unicoq/unicoq"
+bug-reports: "https://github.com/unicoq/unicoq/issues"
+license: "MIT"
+build: [
+  ["coq_makefile" "-f" "_CoqProject" "-o" "Makefile"]
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml"
+  "coq" {>= "8.16" & < "8.17~"}
```

Comment:
```suggestion
  "coq" {>= "8.17" & < "8.18~"}
```

Reply:
Many thanks - I am not sure how this could escape my local tests (I build with a local patch before submitting ...)